<?xml version="1.0"?>
<launch>
​
  <!-- Run the map server -->
  <arg name="map_file" default="$(find turtlebot_transport_flexmansys)/mapas/laboratorio/laboratorio_12012002.yaml"/>
  <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" />
​
  <!--- Run AMCL -->
  <include file="$(find turtlebot_transport_flexmansys)/launch/turtlebot_transport_flexmansys_amcl_laboratorio.launch" />
​
  <!--- Run Move Base -->
  <include file="$(find turtlebot_transport_flexmansys)/launch/turtlebot_transport_flexmansys_move_base_laboratorio.launch" />

</launch>
